daml-lf/spec/value.rst

```diff
@@ -253,7 +253,7 @@ digits of decimal precision.  In other words, in base-10, a number with
 A leading sign, + or -, may be optionally included.  In regular
 expression terms::
 
-  [+-]?[0-9]{0,28}(\.[0-9]{0,10})?
+  [+-]?[0-9]{1,28}(\.[0-9]{1,10})?
```

Comment:
Should this be considered retroactively applicable to value version 1?

Reply:
The current spec is clearly buggy.
For instance it was accepted ``"."`` (the string containing only dot) and ``""`` (empty string). 
Therefore we have to "retroactively" fix the spec. 

I took the intersection between 

- the originally specified reegex ```[+-]?[0-9]{0,28}(\.[0-9]{0,10})?```
- and the regexp that was actually used in the code ```[+-]?\d+(\.\d+)?```

Like that we actually do not accept less correct string representation.

Reply:
yes, me and @remyhaemmerle-da discussed this and we're retroactively applying this as a bug fix.